controllers/scraper_test.go

```diff
@@ -0,0 +1,116 @@
+package controllers_test
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+
+	. "go-crawler-challenge/tests"
+	. "go-crawler-challenge/tests/fixtures"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("ScraperController", func() {
+	BeforeEach(func() {
+		keyword := "keyword"
+		visitURL := fmt.Sprintf("https://www.google.com/search?q=%s&lr=lang_en", url.QueryEscape(keyword))
```

Comment:
How about place "https://www.google.com/search?q=%s&lr=lang_en" into a constant?

Reply:
@junan Can you give me more explanation?
Do you have an example for that? 🤣